app/components/addon-tabs.js

```diff
@@ -1,6 +1,48 @@
 import Component from '@ember/component';
 
+const KEY = {
+  ARROW_LEFT: 'ArrowLeft',
```

Comment:
Since IE11 support is required, this should be converted to KeyCodes instead

Reply:
Changed to use KeyboardEvent.key instead and added support for the IE11 version. Adding a polyfill might be a better solution though.